dataland-frontend/src/components/forms/CreateEUTaxonomy.vue

```diff
@@ -4,14 +4,13 @@
     </template>
     <template #content>
       <FormKit
-        v-model="model"
-        submit-label="Post EU-Taxonomy Dataset"
-        :submit-attrs="{
-                'name': 'postEUData'
-              }"
-        type="form"
-        id="createEuTaxonomyForm"
-        @submit="postEUData">
+          v-model="model"
+          :actions="false"
+          type="form"
+          id="createEuTaxonomyForm"
+          @submit="postEUData"
+          #default="{ state: { valid } }"
+      >
```

Comment:
Kann mir diese Änderungen jmd erklären? (Analog auch in CreateCompany.vue zu finden)

Reply:
Hier geht es darum das der Button zum upload erst dann klickbar wird wenn die Felder befüllt sind. Siehe unten (Zeile 166) "submit" ist "disabled" wenn "!valid".

Reply:
Selbst anschauen nochmal